src/desktop/apple_notification.mm

```diff
@@ -47,40 +32,18 @@ bool available() {
 	return false;
 }
 
-#ifdef HAVE_NS_USER_NOTIFICATION
 void send_cocoa_notification(const std::string& owner, const std::string& message);
-#endif
-#ifdef HAVE_GROWL
-void send_growl_notification(const std::string& owner, const std::string& message, const desktop::notifications::type note_type);
-#endif
 
-#pragma clang diagnostic push
```

Comment:
Don't remove this `#pragma`, or the `pop` lower down.

Reply:
Sure